src/cmd/rawdns/main.go

```diff
@@ -174,7 +196,13 @@ func handleDockerRequest(domain string, w dns.ResponseWriter, r *dns.Msg) {
 			return
 		}
 
-		containerIp := container.NetworkSettings.IpAddress
+		var containerIp string
+		if domain == "swarm." {
```

Comment:
This is a total hack for now, just to get it working.  Would it make sense to add a 'swarm' flag to the config file, and reference it like `config[domain].Swarm`?